webapp/templates/partials/scoreboard_summary.html.twig

```diff
@@ -24,19 +24,19 @@
                     <a {% if link %}href="{{ link }}"{% endif %}>
                         <i class="fas fa-thumbs-up fa-fw"></i>
                         <span style="font-size:90%;" title="number of accepted submissions">
-                                {{ summary.numberOfCorrectSubmissions(sortOrder) }}
+                                {{ summary.numSubmissionsCorrect[sortOrder] }}
```

Comment:
This (and below) crashes if sortOrder is -1, which happens when there are no teams because scoreboard_table.html.twig displays the summary of the last sortorder group even if it is empty:
https://github.com/DOMjudge/domjudge/blob/cea1a256301806c5768e7b3b6faffbd13fd55c51/webapp/templates/partials/scoreboard_table.html.twig#L288:L289
Compare this to the summaries of the earlier sortorder groups, which explicitly check for -1:
https://github.com/DOMjudge/domjudge/blob/cea1a256301806c5768e7b3b6faffbd13fd55c51/webapp/templates/partials/scoreboard_table.html.twig#L108:L110

One way to fix this would be to use `... ?? 0` here like the old getters did.
Another way would be to remove the summary of the last sortorder group if the sortorder is -1, which will omit the summary row when there are no teams (the previous behaviour was to display a summary row with all 0's for the stats).